safe/gis/vector/summary_5_multi_exposure.py

```diff
@@ -228,6 +229,8 @@ def multi_exposure_aggregation_summary(aggregation, intermediate_layers):
 
     aggregation.commitChanges()
     aggregation.keywords['title'] = layer_purpose_aggregation_summary['name']
+    aggregation.keywords['layer_purpose'] = (
+        layer_purpose_aggregation_summary['key'])
```

Comment:
I had issue with this as well, it says `aggregation` in the keywords before

Reply:
Yes I noticed. And I needed this fix too. Thanks for the hint. Git will merge automatically this for you because it's identical in your PR